android/app/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
     defaultConfig {
         applicationId "org.openbot"
         minSdkVersion 21
-        targetSdkVersion 29
+        targetSdkVersion 30
```

Comment:
This needs to be kept at 29, otherwise the logging does not work. There has been a change in storage permissions in API30.